CriteoPublisherSdk/Tests/UnitTests/CriteoTests.m

```diff
@@ -205,6 +205,21 @@ - (void)testLoadBidForAdUnit_GivenNoContext_UseEmptyOne {
                           responseHandler:OCMArg.any]);
 }
 
+- (void)testChildDirectedTreatment {
+
+  // case nil - undefined
+  XCTAssertNil(self.criteo.childDirectedTreatment);
+  XCTAssertNil([self.criteo bidManager].childDirectedTreatment);
+
+  // case false
+  self.criteo.childDirectedTreatment = [NSNumber numberWithBool:false];
```

Comment:
isn't it possible to use `@YES` or `@NO` ?

Reply:
It sure is possible.
Pushed commit: [Changed instantiation of boolean object to @YES and @NO](https://github.com/criteo/ios-publisher-sdk/pull/244/commits/73d6ad7749e2793e242e807f0b9573e3f8321c77)
Resolving this conversation.